va/va.go

```diff
@@ -194,8 +195,8 @@ type verificationRequestEvent struct {
 	Hostname          string                  `json:",omitempty"`
 	ValidationRecords []core.ValidationRecord `json:",omitempty"`
 	Challenge         core.Challenge          `json:",omitempty"`
-	ValidationLatency time.Duration           `json:",omitempty"`
-	Error             string                  `json:",omitempty"`
+	ValidationLatency float64
```

Comment:
You're going to want to gofmt this.

Reply:
I've got vim set to gofmt everything on save - what looks unformatted here?

Reply:
Oops, missed you'd removed the json tag here which caused the below tag to not be indented in line.